chaos/chaos.go

```diff
@@ -107,8 +107,8 @@ func (c *Chaos) terminate(clientset kube.Interface) error {
 
 	killValue, err := c.getKillValue(clientset)
 
-	// KillAll is the only kill type that does not require a kill-value
-	if killType != config.KillAllLabelValue && err != nil {
+	// KillAll and KillPodDisruptionBudget are the only kill types that do not require a kill-value
```

Comment:
Could you leave a comment here reminding us to refactor this logic if we add one more kill type that does not require a kill value. I don't want us to continue adding to this if-condition. For now its ok, but if add more such kill-types, we should make "requires kill value" a property of the kill-type

Reply:
Yeap, done.